crates/core/tedge_mapper/src/sm_c8y_mapper/topic.rs

```diff
@@ -7,6 +7,7 @@ pub(crate) enum IncomingTopic {
     SoftwareListResponse,
     SoftwareUpdateResponse,
     SmartRestRequest,
+    RestartResponse,
 }
 
 impl IncomingTopic {
```

Comment:
These topics must be defined by the agent and used by the mapper.

This will be done in a follow up task: #663.